src/components/checkbox/checkbox.spec.js

```diff
@@ -298,15 +299,137 @@ describe('mdCheckbox', function() {
       expect(isChecked(checkbox)).toBe(false);
     });
 
-    it('should submit a parent form when ENTER is pressed', function () {
-      var form = compileAndLink('<form><md-checkbox ng-checked="value"></md-checkbox></form>');
-      angular.element(form[0].getElementsByTagName('md-checkbox')[0]).triggerHandler({
-        type: 'keypress',
-        keyCode: $mdConstant.KEY_CODE.ENTER
+    it('should not submit a parent form when ENTER is pressed and there is no submit button/input',
```

Comment:
There should also be a case that tests that the checkbox triggers the *first* non-disabled submit button when there are multiple.

Reply:
Added.